sdk/compiler/damlc/tests/BUILD.bazel

```diff
@@ -633,6 +634,10 @@ da_haskell_test(
         "//test-common:upgrades-FailsWhenAnExceptionIsDefinedInAnUpgradingPackageWhenItWasAlreadyInThePriorPackage-v2.dar",
         "//test-common:upgrades-FailsWhenAnInstanceIsReplacedWithADifferentInstanceOfAnIdenticallyNamedInterface-v1.dar",
         "//test-common:upgrades-FailsWhenAnInstanceIsReplacedWithADifferentInstanceOfAnIdenticallyNamedInterface-v2.dar",
+        "//test-common:upgrades-WarnsWhenUsingLF115DependencyInSerializablePositionDep-v1.dar",
+        "//test-common:upgrades-WarnsWhenUsingLF115DependencyInSerializablePositionDep-v2.dar",
+        "//test-common:upgrades-WarnsWhenUsingLF115DependencyInSerializablePosition-files",
+        "//test-common:upgrades-WarnsWhenUsingDamlScriptDatatype-files",
```

Comment:
Add a negative test, i.e. if an LF15 package uses daml-script-lts 1.15 shouldn't warn
Add a negative test, if an LF15 package is referenced in a nonserializable type it shouldn't warn